### ISSUE TITLE
Improve Samples documentation in the website

### ISSUE THREAD

**Issue by harinikmsft** [microsoft/react-native-windows-samples#107]
Right now we have a top header item for Samples that just links to the repo folder. Each sample app today has good Readme files - it may be valuable to have a page for each Sample app in the website that sets context on what each is illustrating and render the readme files in the website?

**Comment by chrisglein** [comment]
Agreed, would be great to have this more discoverable from the website.
